x/batch/msg.go

```diff
@@ -0,0 +1,30 @@
+package batch
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/x"
+)
+
+const (
+	PathExecuteBatchMsg = "batch/execute"
+)
+
+type Msg interface {
+	weave.Msg
+	x.Validater
+	MsgList() ([]weave.Msg, error)
+}
+
+func Validate(msg Msg) error {
```

Comment:
q: why not have this on the application level where you call this from? That would also help with the `MaxBatchMessages` as this limitation can be application specific.